demo/pom.xml

```diff
@@ -62,67 +62,73 @@
 
     <!--***START temporary override of versions*** -->
     <!-- ***END temporary override of versions*** -->
-
-    <gallery-magick.version>${project.version}</gallery-magick.version>
-
-    <essentials.version>13.0.0</essentials.version>
-    <jsp-api.version>2.2</jsp-api.version>
-    <taglibs.version>1.2.5</taglibs.version>
+    <forge.gallery-magick.version>${project.version}</forge.gallery-magick.version>
+    <essentials.version>16.0.0</essentials.version>
```

Comment:
This could be updated to <essentials.version>${hippo.release.version}</essentials.version> so that it will get updated to match the parent version when that is updated.